internal/usecase/interfaces/published.go

```diff
@@ -19,10 +19,23 @@ type ProjectPublishedMetadata struct {
 }
 
 func ProjectPublishedMetadataFrom(prj *project.Project) ProjectPublishedMetadata {
```

Comment:
Is there any reason that this function is declared here since this file usually has only type and interfaces. I assume this function can be moved from this file to `interactor/published.go`. 
@rot1024 How do you think about this?

Reply:
No problem.

`ProjectPublishedMetadata` is defined at `interfaces` package. `ProjectPublishedMetadataFrom` is operation using ProjectPublishedMetadata and does not have any dependency on other uppers layers. That's why it can be treated as a `interfaces`.